mmv1/templates/terraform/schema_property.erb

```diff
@@ -111,20 +112,26 @@
         },
       },
     <% end -%>
-  <% elsif property.item_type.is_a?(String) # Basic type like Api::Type::String -%>
-      Elem: &schema.Schema{
-        Type: <%= tf_types[property.item_type_class] -%>,
-      },
-  <% elsif property.item_type.is_a?(Api::Type::Enum) -%>
-      Elem: &schema.Schema{
-        Type: <%= tf_types[property.item_type.class] -%>,
-        ValidateFunc: verify.ValidateEnum([]string{"<%= property.item_type.values.join '","' -%>"}),
-      },
   <% else # array of basic types -%>
       Elem: &schema.Schema{
+    <% if property.item_type.is_a?(String) # Basic type like Api::Type::String -%>
```

Comment:
Am I missing something here, or are both results in this if/else block identical?

Reply:
they're different - the first one is `item_type_class` and the second one is `item_type.class` (with a period between type and class instead of an underscore).

Reply:
There it is, thanks!